clients/src/main/java/org/apache/kafka/common/requests/ProduceResponse.java

```diff
@@ -67,14 +68,19 @@
      * INVALID_PRODUCER_EPOCH (47)
      * CLUSTER_AUTHORIZATION_FAILED (31)
      * TRANSACTIONAL_ID_AUTHORIZATION_FAILED (53)
+     * INVALID_RECORD (86)
```

Comment:
Can you move these to the class doc and add links? See OffsetFetchResponse for an example.